### DISCUSSION TITLE
Can I keep the format when copying data by use-clipboard ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4372] [Q&A]
expected 
<img width="366" alt="image" src="https://github.com/mantinedev/mantine/assets/124666577/83f5ea14-df78-4e53-84ad-8f34c95b6796">

```jsx
   clipboard.copy(
                    `name      age      address
                    zhangsan   12.      abcdefg`
                  )
```

current
<img width="466" alt="image" src="https://github.com/mantinedev/mantine/assets/124666577/2b174181-f487-42b9-a4be-a6c8d0a02f5f">

**Comment by rtivital** [comment]
use-clipboard uses navigator.clipboard.writeText. It copies the exact string that you've specified, including all white space. Hook does not modify value that you've passed to the copy function.
